lib/open_project/openid_connect/engine.rb

```diff
@@ -50,6 +50,19 @@ def configuration
       end
     end
 
+    #config.to_prepare do
+    initializer 'openid_connect.form_post_method' do
+      # If response_mode 'form_post' is chosen,
+      # the IP sends a POST to the callback. Only if
+      # the sameSite flag is not set on the session cookie, is the cookie send along with the request.
+      if OpenProject::Configuration.openid_connect.any? { |_, v| v['response_mode']&.to_s == 'form_post' }
+        SecureHeaders::Configuration.default.cookies[:samesite][:lax] = false
```

Comment:
Is there no way to disable this just for the `/auth/<provider>` requests?
Those are the only ones who redirect to the authentication provider who does the form post.

Reply:
No, unfortunately not. We could restrict it to just be applied to the session cookie but then again, it is the only cookie we write. Alternatively, we could potentially create a separate cookie until we are logged in and work with that.